css/style.css

```diff
@@ -25,29 +26,121 @@ ul {
   font-size: 30px;
 }
 
+/* Mobile Menu  */
+
+.nav-bar ul {
+  display: none;
+}
+
 li {
   padding: 10px;
+  list-style: none;
 }
 
-nav {
+a {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--sub2k);
+}
```

Comment:
- [ ] Kindly make sure your mobile menu design is spaced as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/179791037-d3744dc9-62d7-4ae0-a48c-206abb4a8225.png)  |  ![image](https://user-images.githubusercontent.com/74814780/179791068-a39dd53f-0b3f-40e0-ac7d-6aaf633d6cce.png)